assets/scss/base/_icons.scss

```diff
@@ -114,6 +114,29 @@ Styleguide Icons.Exclamation
 }
 
 /*
+Tick
+
+The tick icon is used to highlight actioned content to the user.
+
+Markup:
+<i class="tick"></i>
```

Comment:
Hi Adam, agreed. 

Something like this should do the trick for screen readers:

`<span class="tick" role="img" aria-label="tick"></span>`

I'll update the pr in 30 mins or so